spec/mailers/candidate_mailer_offers_and_rejections_spec.rb

```diff
@@ -146,6 +146,29 @@
           expect(email.body).not_to include('Future applications')
         end
       end
+
+      context 'when it is before the apply_2_deadline' do
+        before do
+          allow(EndOfCycleTimetable).to receive(:between_cycles_apply_2?).and_return(false)
+        end
+
+        it 'informs the candidate they can apply again this year' do
+          expect(email.body).to include('If now’s the right time for you, you can still apply for teacher training again this academic year.')
+        end
+      end
+
+      context 'when it is after the apply_2_deadline' do
+        before do
+          allow(EndOfCycleTimetable).to receive(:between_cycles_apply_2?).and_return(true)
+          allow(EndOfCycleTimetable).to receive(:apply_reopens).and_return(Date.new(2021, 10, 13))
+          allow(RecruitmentCycle).to receive(:cycle_name).and_return('2021 to 2022')
```

Comment:
Would it work if you just stubbed `RecruitmentCycle.current_year`?

Reply:
Yea sorry i messed up and knew it was meant to return the cycle then did a diff implementation 🤦